exporter/awsprometheusremotewriteexporter/auth.go

```diff
@@ -100,6 +106,17 @@ func getCredsFromConfig(auth AuthConfig) *credentials.Credentials {
 	return creds
 }
 
+var regionRe = regexp.MustCompile(`aps-workspaces\.(.*)\.amazonaws\.com`)
```

Comment:
Dunno if it matters but I'd usually explicitly exclude dot for safety (?)

`[^.]+`

Either way plus seems nicer than asterisk since we expect something to always be there.

Reply:
Obsolete now.